Add a note for hierarchy levels #1948 (#2018)
IgniteUI/igniteui-docfx: en/components/treegrid/tree_grid.md

|Templating Tree Cells|When templating a tree cell, content that spans outside the boundaries of the cell will not be shown unless positioned in an overlay.|
|Group By|Group By feature is not supported, because it is inherent to the tree grid.|

> [!NOTE]

> > The tree grid has a depth limit of 25 levels. Supporting bigger hierarchies requirers adding custom classes. You may see an example of the workaround below:

Reviewer: ```suggestion
> The tree grid has a depth limit of 25 levels. Supporting more levels requires adding custom CSS classes in the application. You may see an example of such CSS class below:
```
